framework/packages/abstract-testing/src/lib.rs

```diff
@@ -117,11 +118,23 @@ pub fn abstract_mock_querier(mock_api: MockApi) -> MockQuerier {
     abstract_mock_querier_builder(mock_api).build()
 }
 
+/// cosmwasm_std::mock_env_validated(deps.api), but address generated with MockApi
+pub fn mock_env_validated(mock_api: MockApi) -> Env {
```

Comment:
We should open an issue on cosmwasm to change the env.contract.addr to a valid addr instead of the string that it currently is.

Reply:
Yes ! Just like they removed mock_info to message_info !